packages/core/docs/commercetools/use-review.md

```diff
@@ -34,6 +37,26 @@ interface ReviewDraft {
   rating: number;
 }
 ```
+
+:::
+
+::: slot usage
+
+## Usage
+
+When `@vsf-enterprise/ct-reviews` plugin is installed as a dependency, it requires minor modifications in the code to work.
```

Comment:
@Fifciu @filrak don't you guys think that doc should be in the enterprise repo?

Reply:
That's an excellent question and honestly I don't know. I thought about this and it is far more complex topic than it may seem. Let's discuss it on a call ;)